instrumentation/grpc-1.6/javaagent/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ tasks {
     jvmArgs("-Dotel.instrumentation.grpc.capture-metadata.client.request=some-client-key")
     jvmArgs("-Dotel.instrumentation.grpc.capture-metadata.server.request=some-server-key")
     jvmArgs("-Dotel.instrumentation.common.experimental.controller-telemetry.enabled=true")
+    jvmArgs("-Djava.net.preferIPv4Stack=true")
```

Comment:
can you add comment why?

is it just so we can verify `network.type=ipv4`?

Reply:
It is `network.type` and `network.peer.address`. Seems to happen only in latest dep tests and not always.